ddocs/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -4,7 +4,7 @@ function(doc) {
   var usedKeys = [];
   var emitMaybe = function(key, value) {
     if (usedKeys.indexOf(key) === -1 && // Not already used
-        !key.match(/(^$)|(^[^A-Za-z0-9+])/) && // Not empty or starting with bad symbol
+        !key.match(/(^$)/) && // Not empty
```

Comment:
This is a very fun way to check if a string is not empty.  Perhaps more traditional to write `key`.  But the line should probably just be deleted, because the next line is `key.length > 2`.